Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM alpine:3.12.7
 RUN apk add --no-cache util-linux
+RUN apk add xfsprogs xfsprogs-extra
```

Comment:
can you please add above two packages after `util-linux`? 

The lines can look like:
```
FROM alpine:3.12.7
RUN apk add --no-cache util-linux xfsprogs xfsprogs-extra
```

This will reduce creating additional layers and also help with keeping the size of the image smaller.